### PR TITLE
Python: fix typos, and other copyedits

### PR DESCRIPTION
**Pull request by chalin**
cc @ejona86 @jtattermusch

### CODE REVIEW

File: content/docs/languages/python/generated-code.md [hunk 1]
```diff
@@ -10,13 +11,13 @@ gRPC Python relies on the protocol buffers compiler (`protoc`) to generate
 code.  It uses a plugin to supplement the generated code by plain `protoc`
 with gRPC-specific code.  For a `.proto` service description containing
 gRPC services, the plain `protoc` generated code is synthesized in
-a `_pb2.py` file, and the gRPC-specifc code lands in a `_grpc_pb2.py` file.
-The latter python module imports the former.  In this guide, we focus
+a `_pb2.py` file, and the gRPC-specific code lands in a `_pb2_grpc.py` file.
```
Comment (chalin) [inline content/docs/languages/python/generated-code.md:14]: In particular, note the fix here: `_grpc_pb2.py` --> `_pb2_grpc.py`

**Comment by ejona86** [comment]
@gnossen, could you review?

File: content/docs/languages/python/generated-code.md [hunk 2]
```diff
@@ -32,8 +33,8 @@ service FortuneTeller {
 }
 ```
 
-gRPC `protoc` plugin will synthesize code elements along the lines
-of what follows in the corresponding `_pb2_grpc.py` file:
+When compiled, the gRPC `protoc` plugin generates code similar to the following
```
[the remark this answers is not in the document]
Comment (chalin) [reply content/docs/languages/python/generated-code.md:36]: Oh, right you are! Thanks for catching that. 

I'll fix it to read "When the service is compiled, ...", commit and then merge.

**Comment by chalin** [comment]
Thank you for the review @gnossen.

**Pull request merged**